modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/operations/util/OperationReply.java

```diff
@@ -686,38 +512,268 @@ private Item appendReplaceInfo(Operation operation) throws JSONException {
         return null;
     }
 
-    public Exec executeCommand(Operation operation) {
-        Exec execElement = new Exec();
+    public ExecuteTag executeCommand(Operation operation) {
+        ExecuteTag execElement = new ExecuteTag();
         execElement.setCommandId(operation.getId());
-        List<Item> itemsExec = new ArrayList<Item>();
+        List<Item> itemsExec = new ArrayList<>();
         Item itemExec = appendExecInfo(operation);
         itemsExec.add(itemExec);
         execElement.setItems(itemsExec);
         return execElement;
     }
 
-    public Sequence buildSequence(Operation operation, Sequence sequenceElement) {
+    public Sequence buildSequence(Operation operation, Sequence sequenceElement) throws WindowsOperationException,
+            JSONException {
+
         sequenceElement.setCommandId(operation.getId());
+        List<Replace> replaceItems = new ArrayList<>();
+
+        if (operation.getCode().equals(PluginConstants.OperationCodes.LOCK_RESET)) {
+            ExecuteTag execElement = executeCommand(operation);
+            Get getElements = new Get();
+            getElements.setCommandId(operation.getId());
+            List<Item> getItems = new ArrayList<>();
+            Item itemGets = appendGetInfo(operation);
+            getItems.add(itemGets);
+            getElements.setItems(getItems);
+
+            sequenceElement.setExec(execElement);
+            sequenceElement.setGet(getElements);
+            return sequenceElement;
+
+        } else if (operation.getCode().equals(PluginConstants.OperationCodes.POLICY_BUNDLE)) {
+            List<? extends Operation> policyOperations;
+            try {
+                policyOperations = (List<? extends Operation>) operation.getPayLoad();
+            }catch (ClassCastException e) {
+                throw new ClassCastException();
+            }
+            for (Operation policy : policyOperations) {
+                if (policy.getCode().equals(PluginConstants.OperationCodes.CAMERA)) {
+                    Replace replaceCameraConfig = new Replace();
+                    Item cameraItem;
+                    List<Item> cameraItems = new ArrayList<>();
+                    try {
+                        cameraItem = appendReplaceInfo(policy);
+                        cameraItems.add(cameraItem);
+                    } catch (JSONException e) {
+                        throw new WindowsOperationException("Error occurred while parsing payload object to json.");
+                    }
+                    replaceCameraConfig.setCommandId(operation.getId());
+                    replaceCameraConfig.setItems(cameraItems);
+                    replaceItems.add(replaceCameraConfig);
+                }
+                if (policy.getCode().equals(PluginConstants.OperationCodes.ENCRYPT_STORAGE)) {
```

Comment:
As a best practice, always compare the non-deterministic variable values against known constants.